Specs/DateTime/Chinese/DateTimeModel.json

```diff
@@ -3921,13 +3920,12 @@
     "Context": {
       "ReferenceDateTime": "2019-01-06T12:00:00"
     },
-    "Comment": "This is notsupported for now because of the FromToRegex",
-    "NotSupported": "dotnet, java, javascript, python",
+    "NotSupported": "java, javascript, python",
     "Results": [
       {
-        "Text": "自2014年6月14日",
-        "Start": 5,
-        "End": 15,
+        "Text": "2014年6月14日起",
```

Comment:
Same here. Can you explain the difference? I don't speak Chinese. :-(

Reply:
“自2014年6月14日起” is like "from 2014-6-14 begin", the ideal result is the whole sentence. But because of the issue related to PotentialAmbiguousRangeRegex, I did a workaround to get part of the result. “自2014年6月14日起” and  "2014年6月14日起" mean the same in chinese.